assets/js/googlesitekit/api/index.test.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Internal dependencies
+ */
+import API, { invalidateCache, get, set, useCache } from './index';
+
+describe( 'googlesitekit.api', () => {
+	it( 'should have caching enabled by default', () => {
+		expect( API._useCache ).toEqual( true );
```

Comment:
See above, here we should then use `useCache()` in "get mode", not directly access the variable.